backend/src/test/scala/com/softwaremill/bootzooka/user/UserApiTest.scala

```diff
@@ -168,6 +168,23 @@ class UserApiTest extends BaseTest with Eventually with TestDependencies with Te
     requests.getUser("invalid").code shouldBe StatusCode.Unauthorized
   }
 
+  "/user/logout" should "logout the user" in {
```

Comment:
shouldn't we also add a test which checks that changing the password logs out of every session?